deepforest/visualize.py

```diff
@@ -257,37 +257,225 @@ def label_to_color(label):
     return color_dict[label]
 
 
-def convert_to_sv_format(df):
+def convert_to_sv_format(df, width=None, height=None):
     """Convert DeepForest prediction results to a supervision Detections
     object.
 
     Args:
         df (pd.DataFrame): The results from `predict_image` or `predict_tile`.
                            Expected columns: ['xmin', 'ymin', 'xmax', 'ymax', 'label', 'score', 'image_path'].
+        width (int): The width of the image in pixels. Required if the geometry type is 'polygon'.
+        height (int): The height of the image in pixels. Required if the geometry type is 'polygon'.
 
     Returns:
         sv.Detections: A supervision Detections object containing bounding boxes, class IDs,
```

Comment:
Looks like for point data we return sv.KeyPoints, so flagging to make applicable to all output types